.github/workflows/build.yml

```diff
@@ -40,3 +40,9 @@ jobs:
           context: .
           push: true
           tags: ${{ steps.login-ecr-vaec.outputs.registry }}/notification_api:${{ inputs.ref }}_${{ inputs.environment }}
+
+      - name: Dispatch Twistlock Workflow
+        env:
+          GH_TOKEN: ${{ github.token }}
```

Comment:
Where is it getting that token from?

Reply:
I think it's a built in token. 
This [action doc](https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows) shows it getting it from secrets.GITHUB_TOKEN, however:

github.token is a secret token that is automatically provided by GitHub to GitHub Actions, which allows the action to authenticate and interact with the GitHub API on behalf of the repository or organization.

Here is a link to the official GitHub documentation that explains the github.token and other built-in environment variables that can be used in GitHub Actions:

[Context and expression syntax for GitHub Actions](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context)
In particular, you may be interested in the section on github.token specifically:

[Authentication in a workflow](https://docs.github.com/en/actions/reference/authentication-in-a-workflow)
This section explains how github.token is generated and how it can be used in GitHub Actions to authenticate with the GitHub API.

Reply:
That's great that there's a built-in token. We were unaware of that before and Jesse made a token he added to the secrets, seemed like the only option at the time.